### DISCUSSION TITLE
Allow specifying an intermediate file in GitHub actions

### DISCUSSION THREAD

**Discussion by GytisBraz** [ossillate-inc/packj#20] [Ideas]
As far as I understand, `packj` GitHub action expects the dependency file to already be in the project's repository https://github.com/ossillate-inc/packj-github-action/blob/f99256ef53d2e9e1fd9776a07535219f20b087b3/action.yaml#L41

My use case is a little bit different - I am generating a `requirements.txt` file from another format (poetry) and want to pass that generated file into `packj`. However, this is not possible at the moment because the dependency file is expected to be in the repo, whereas I am generating it in an intermediate GH actions step.

Is this something worth considering to support? Are pull requests welcome (I could update the GitHub action definition if need be)?

**Comment by ashishbijlani** [comment]
Yes, this is something that should be supported. We'd deeply appreciate your code contribution to enable this use case. Feel free to create a PR. Thanks!

**Reply by ashishbijlani** [reply]
Hi @GytisBraz , just wondering if you're working on this and plan to send your changes?

**Reply by GytisBraz** [reply]
Hey, I will try to get at it this weekend, sorry for the long silence 😁

**Reply by ashishbijlani** [reply]
Sounds good. Thanks @GytisBraz !

**Reply by GytisBraz** [reply]
PR is ready :pray: https://github.com/ossillate-inc/packj-github-action/pull/5 :pray:
